### PR TITLE
Update jupyterhub-internal to latest helm chart, add jupyterhub-itr

### PR DESCRIPTION
**Pull request by manics**
~This includes an upgrade to JupyterHub 0.9 which is backward compatible with 0.8 images, and is intended to help us test whether our notebook images can be easily updated to JupyterHub 0.9.~

This PR takes advantage of new features to provide a dropdown of images, allowing this jupyterhub instance to support multiple notebook images. This means https://ome-lochy.openmicroscopy.org/jupyterhub-training could be removed in future.

screen shot 2018-06-29 at 17 06 39-fullpage

## Testing notes

https://ome-lochy.openmicroscopy.org/jupyterhub-internal

Also adds https://ome-lochy.openmicroscopy.org/jupyterhub-itr

### CODE REVIEW

**Comment by manics** [comment]
I've added https://ome-lochy.openmicroscopy.org/jupyterhub-itr/ which uses `snoopycrimecop/idr-notebooks:itr_merge_trigger`

**Comment by jburel** [comment]
Installation works nicely.
Is there a UI option to have the images in a combobox?

**Comment by manics** [comment]
There are no UI options, other than changing the display name for each image.

File: jupyterhub-internal/zero-to-jupyterhub-config.yml [hunk 1]
```diff
@@ -3,6 +3,54 @@ hub:
...
+            'snoopycrimecop/training-notebooks:master_merge_daily',
+        }
+      }, {
+        'display_name': 'openmicroscopy/training-notebooks:latest',
+        'kubespawner_override': {
+          'singleuser_image_spec':
+            'openmicroscopy/training-notebooks:latest',
+        }
+      }, {
+        'display_name': 'manics/idr-notebooks:devel',
```
Comment (jburel) [inline jupyterhub-internal/zero-to-jupyterhub-config.yml]: you may want to remove yours from that list

File: jupyterhub-internal/zero-to-jupyterhub-config.yml [hunk 2]
```diff
@@ -20,13 +68,18 @@ singleuser:
     tag: master_merge_trigger
     pullPolicy: Always
   startTimeout: 1800
+  cpu:
+    limit: 2
+    guarantee: 0.1
   memory:
-    limit: 1G
-    guarantee: 100M
+    limit: 2G
```
Comment (jburel) [inline jupyterhub-internal/zero-to-jupyterhub-config.yml]: as we have seen especially with some of the ITR notebooks 
we will need more than 2G

File: jupyterhub-itr/zero-to-jupyterhub-config.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ingress:
+  enabled: true
+  hosts:
+  - ome-lochy.openmicroscopy.org
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    ingress.kubernetes.io/proxy-body-size: 16m
+    ingress.kubernetes.io/proxy-read-timeout: 3600
+    ingress.kubernetes.io/proxy-send-timeout: 3600
+    #kubernetes.io/tls-acme: 'true'
```
Comment (jburel) [inline jupyterhub-itr/zero-to-jupyterhub-config.yml]: Need to keep that comment?

**Comment by jburel** [comment]
The description of the PR could be updated since jupyterhub has not been upgraded to 0.9

File: jupyterhub-internal/zero-to-jupyterhub-config.yml (hunk 1 above)
Comment (manics) [reply jupyterhub-internal/zero-to-jupyterhub-config.yml]: Done

**Comment by jburel** [comment]
Changes look good
Ready to merge

**Comment by jburel** [comment]
in ``jupyterhub-internal/README.md`` and ``jupyterhub-itr/README.md``
the ``jupyterhub/jupyterhub`` version are different 
I understood they were the same

**Comment by manics** [comment]
They're currently different since the itr work needed a newer version. I'll open a separate cleanup PR later.

**Comment by jburel** [comment]
Merging

**Pull request merged**